packages/notification-spa/src/app/header/header.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="breadcrumbs">
```

Comment:
Change the class names as well. This `div` is the header.
The `div` below this line where the actual breadcrumb starts.